### ISSUE TITLE
ma-sp.tum.sexy doesn't seem to work

### ISSUE THREAD

**Issue by mammuth** [mammuth/TUM.sexy#96]
Redirects to _http://tum.sexy_
```
$ curl -I ma-sp.tum.sexy                                                                      
HTTP/1.1 302 Found
Date: Wed, 07 Feb 2018 16:11:54 GMT
Server: Apache/2.4.25 (Debian)
Location: http://tum.sexy/
Content-Type: text/html; charset=UTF-8
```

Route exists apparently.

**Comment by MW360** [comment]
fixed it in pull request #135

**Comment by kordianbruck** [comment]
Works now - thanks @MW360
